tests/settings.test.ts

```diff
@@ -140,6 +82,9 @@ describe.each([{ permission: 'Master' }, { permission: 'Admin' }])(
         },
         faceting: {
           maxValuesPerFacet: 50,
+          sortFacetValuesBy: {
+            '*': 'alpha' as 'alpha',
```

Comment:
Why do you need the alpha as alpha?

Reply:
I need to tell Typescript that this value is not mutable. By saying `as alpha` I'm confirming typescript that this value will remain `alpha`. I have a few possibilities to tackle this issue:

<img width="554" alt="Screenshot 2023-07-10 at 15 45 32" src="https://github.com/meilisearch/meilisearch-js/assets/33010418/b2217696-01ce-4ad7-a346-e6167397348e">


I hesitated with the last one `test3`. I left the `as alpha` to help users find a loophole if they come across the same type error.

Nonetheless, I'm more convinced about test3 readability

Reply:
Thanks for the lesson! :)